### ISSUE TITLE
Multipart-formdata not supported with MULTIPLE_BODY_PARAMETERS

### ISSUE THREAD

**Issue by alexhcheng** [Azure/oav#805]
following the spec here: https://swagger.io/docs/specification/2-0/file-upload/

snippet of swagger
```
   "/MyMethod": {
      "post": {
        "summary": "test",
        "description": "test",
        "operationId": "MyOperationId",
        "consumes": [
          "multipart/form-data"
        ],
        "produces": [
          "application/json"
        ],
        "parameters": [
          {
            "in": "query",
            "name": "q1",
            "description": "test",
            "type": "string"
          },
          {
            "in": "query",
            "name": "q2",
            "description": "test",
            "required": true,
            "type": "string"
          },
          {
            "in": "formData",
            "name": "Document",
            "description": "Actual Document in the form",
            "required": true,
            "type": "file"
          },
          {
            "in": "formData",
            "name": "Glossary",
            "description": "Glossary for the request",
            "type": "file"
          }
        ],
```

Running the validation gives MULTIPLE_BODY_PARAMETERS error

**Comment by alexhcheng** [comment]
swagger renders properly in swagger UI

**Comment by raych1** [comment]
This is known issue and we plan to fix it in this sprint.

**Comment by alexhcheng** [comment]
Just wondering when the release date will be so i can report back to my team.  thanks! @raych1

**Comment by raych1** [comment]
@alexhcheng , it's planned to release by end of this month. One thing want to confirm with you, your swagger example above has two formData parameters with `file` type. It's not supported from the openapi document https://swagger.io/docs/specification/2-0/file-upload/. Quote of `However, uploading an arbitrary number of files (an array of files) is not supported`. 

For multiple formData parameters, only one parameter can be `file` type.

**Comment by alexhcheng** [comment]
Hi @raych1, arbitrary number of files are not supported.  But fixed number of files are.  According to the spec.  In the link you sent, under the "Multiple Upload" Section, it states: "You can have several named file parameters, each defined individually".  

This has multiple "file" type, each with different "name".  This format needs to be supported.
```
      parameters:
        - in: formData
          name: upfile1
          type: file
          required: true
        - in: formData
          name: upfile2
          type: file
          required: false
        - in: formData
          name: upfile3
          type: file
          required: false
```

**Comment by ankhspirit** [comment]
@alexhcheng , This issue was resolved in version 3.0.1

**Comment by ankhyk** [comment]
@alexhcheng , This issue was resolved in version 3.0.1
